docs/getting-started/locally.md

```diff
@@ -5,7 +5,6 @@ The cloud is an easy way to get the nio Platform up and running, but to see the
 >**[info] Requirements**
 >
 >* **Python3**: https://www.python.org/downloads (you need Python 3.4+)
->* **Git**: https://git-scm.com/downloads (nio uses git to manage projects)
```

Comment:
This is a requirement, right? Why remove it?

Reply:
The pre-installed git on macOS is sufficient.